input/pagecontent/index.md

```diff
@@ -1134,7 +1144,7 @@ Not all operators are valid for any code system. The `property` and `value` are
 ```
 
 
-#### Defining Code Systems
+#### 👶 Defining Code Systems
```

Comment:
It looks like above you gave Code Systems the 🚧, and that is correct. Using the `CodeSystem` keyword is not yet implemented in SUSHI.

Reply:
fixed.